package.json

```diff
@@ -5,24 +5,46 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "serve" : "webpack serve --mode development",
-    "build" : "webpack --mode production"
+    "serve": "webpack serve --mode development",
+    "build": "webpack --mode production",
+    "lint": "eslint --ext .js,.jsx .",
+    "lint:fix": "eslint --fix --ext .js,.jsx .",
+    "prepare": "husky install"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "echo \"[Husky] pre-commit\""
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "npm run lint:fix",
+      "git add"
```

Comment:
using git add here might cause some changes that you're not intended to commit to be committed. Use just npm run lint instead

Reply:
I removed lint-staged alltogether, no reason having it